server/base/src/main/java/org/apache/accumulo/server/util/ServiceStatusCmd.java

```diff
@@ -48,6 +52,8 @@ public class ServiceStatusCmd {
 
   private static final Logger LOG = LoggerFactory.getLogger(ServiceStatusCmd.class);
 
+  private static final Gson gson = new Gson();
```

Comment:
Can you use `LazySingletons` here or is that unreachable?

Reply:
Fixed in a3a9ddb903